cmd/root.go

```diff
@@ -34,8 +35,15 @@ func CreateConfig() {
 	flag.Parse()
 }
 
-// RunCmdRoot runs the main logic.
+// RunCmdRoot runs the main logic and blocks forever.
 func RunCmdRoot() {
+	requestsSent := safe.DoAndReturn(run, 0)
+	postProcess(requestsSent)
+	block()
+}
+
+// Runs the main logic and returns the number of warmup requests actually sent.
```

Comment:
```suggestion
// run runs the main logic and returns the number of warmup requests actually sent.
```

Reply:
Minor but there's a convention in Go based on which documentation of functions should start with the name of the function.
This is in order to play well with godoc: https://blog.golang.org/godoc :)